src/modules/launcher/PowerLauncher/ViewModel/ContextMenuItemViewModel.cs

```diff
@@ -15,11 +15,50 @@ public class ContextMenuItemViewModel : BaseModel
 
         public string PluginName { get; set; }
 
-        public string Title { get; set; }
+        private string _title;
 
-        public string Glyph { get; set; }
+        public string Title
+        {
+            get => _title;
+            set
+            {
+                if (_title != value)
+                {
+                    _title = value;
+                    OnPropertyChanged(nameof(Title));
+                }
+            }
+        }
+
+        private string _glyph;
+
+        public string Glyph
+        {
+            get => _glyph;
+            set
+            {
+                if (_glyph != value)
+                {
+                    _glyph = value;
+                    OnPropertyChanged(nameof(Glyph));
+                }
+            }
+        }
+
+        private string _fontFamily;
 
-        public string FontFamily { get; set; }
+        public string FontFamily
+        {
+            get => _fontFamily;
+            set
+            {
+                if (_fontFamily != value)
+                {
+                    _fontFamily = value;
+                    OnPropertyChanged(nameof(FontFamily));
+                }
+            }
+        }
```

Comment:
The Accelerator / modifiers and other properties, do those need the OnPropertyChange adjustments as well?

Reply:
I checked the others properties and it seems they don't need it, actually there are properties that should not even have a public set.

Reply:
Creating a bug then to audit that.  Then lets get this merged in.